Xamarin.Forms.Platform.iOS/Renderers/ShellTableViewController.cs

```diff
@@ -59,22 +66,16 @@ void UpdateVerticalScrollMode()
 			switch (_context.Shell.FlyoutVerticalScrollMode)
 			{
 				case ScrollMode.Auto:
-					var pathToFirstRow = Foundation.NSIndexPath.FromRowSection(0, 0);
-					var firstCellRect = TableView.RectForRowAtIndexPath(pathToFirstRow);
-					var firstCellIsVisible = TableView.Bounds.Contains(firstCellRect);
-
-					var lastRowIndex = NMath.Max(0, TableView.NumberOfRowsInSection(0) - 1);
-					var pathToLastRow = Foundation.NSIndexPath.FromRowSection(lastRowIndex, 0);
-					var cellRect = TableView.RectForRowAtIndexPath(pathToLastRow);
-					var lastCellIsVisible = TableView.Bounds.Contains(cellRect);
-
-					TableView.ScrollEnabled = !firstCellIsVisible || !lastCellIsVisible;
+					TableView.ScrollEnabled = true;
+					TableView.AlwaysBounceVertical = false;
```

Comment:
@paymicro is this ok?   From what I can tell your changes broke scrolling when set to Auto